source/_docs/energy.markdown

```diff
@@ -21,3 +21,22 @@ Home Assistant is an open platform and so home energy management is not restrict
 If you have a sensor that returns instantaneous power readings (W or kW), then to add a sensor that returns energy usage or generation (kWh) refer to [Riemann sum integral integration](/integrations/integration/#energy)
 
 <img src='/images/docs/energy/energy-overview.png' alt='Visual representation of how all different energy forms relate.' style='border: 0;box-shadow: none;'>
+
+## Troubleshooting missing entities 
+
+### Condition
+
+You are trying to add a sensor to the energy dashboard, but it does not appear in the selection list. 
+
+### Remedy
```

Comment:
The part below is not a remedy... it only shows what one can do to figure what is wrong.

Reply:
There are a few remedies there, like running the "Fix Issue" flow in statistics, or fixing the integration/template to have the correct value. 

This line was specifically requested to be added by c0ffeeca7, do you maybe have a suggested alternative verbiage that you would prefer?

Reply:
Talked about it with @c0ffeeca7: "Resolution" is the general term we want to adopt for this.